src/components/SubTask/components/SubTaskList/components/subTaskItem/index.js

```diff
@@ -100,10 +111,10 @@ class SubTaskItem extends Component {
                 role="button"
                 tabIndex={0}
                 onClick={() => {
-                  this.changeState(index, subTask, task, tasks);
+                  this.changeState(subTask, task, tasks);
                 }}
                 onKeyDown={() => {
-                  this.changeState(index, subTask, task, tasks);
+                  this.changeState(subTask, task, tasks);
```

Comment:
Could you revert these changes? I added those to get rid of calculating the subtask index, and fix issues with having same subtask name